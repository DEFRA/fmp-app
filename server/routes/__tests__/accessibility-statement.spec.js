const { submitGetRequest } = require('../../__test-helpers__/server')

const constants = require('../../constants')

const url = constants.routes.ACCESSIBILITY_STATEMENT

describe('cookies', () => {
  it('Should return accessibility statement page', async () => {
    const response = await submitGetRequest({ url }, 'Accessibility statement for flood map for planning')
    expect(response.result).toMatchSnapshot()
  })
})
