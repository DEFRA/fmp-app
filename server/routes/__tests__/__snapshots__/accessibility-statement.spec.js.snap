// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cookies Should return accessibility statement page 1`] = `
"<!DOCTYPE html>
<html lang="en" class="govuk-template govuk-template--rebranded">
  <head>
    <meta charset="utf-8">
    <title>
  Accessibility statement for flood map for planning - Flood map for planning - GOV.UK
</title>
    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
    <meta name="theme-color" content="#1d70b8">

    
      <link rel="icon" sizes="48x48" href="/assets/images/favicon.ico">
      <link rel="icon" sizes="any" href="/assets/images/favicon.svg" type="image/svg+xml">
      <link rel="mask-icon" href="/assets/images/govuk-icon-mask.svg" color="#1d70b8">
      <link rel="apple-touch-icon" href="/assets/images/govuk-icon-180.png">
      <link rel="manifest" href="/assets/manifest.json">
    

    
  <meta property="fb:app_id" content="replace_this"/>
  <meta property="og:type" content="website"/>
  <meta property="og:url" content="http://dummyuri/accessibility-statement"/>
  <meta property="og:title" content="Accessibility statement for flood map for planning - Flood map for planning - GOV.UK"/>
  <meta property="og:description" content="Get flood risk information and maps for planning applications and planning permission in England, including flood zones and flood risk assessment data."/>
  <meta property="keywords" content="flood,mapfor,planning,service,gov,uk"/>
  <meta property="description" content="Get flood risk information and maps for planning applications and planning permission in England, including flood zones and flood risk assessment data."/>
  <meta name="google-site-verification" content="hjukuejt64a0rbduxxtbniknb4_cz3x3hlxoxk4_ox8"/>
  
    <meta name="robots" content="noindex"/>
  
  <link href="/assets/application.css" rel="stylesheet"/>

    
  </head>
  <body class="govuk-template__body">
    <script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
    
  




<div class="govuk-cookie-banner js-cookies-banner govuk-!-display-none" data-nosnippet role="region" aria-label="Cookies on Flood Map For Planning">
  
  <div class="govuk-cookie-banner__message js-question-banner govuk-width-container" hidden>

    <div class="govuk-grid-row">
      <div class="govuk-grid-column-two-thirds">
        
        <h2 class="govuk-cookie-banner__heading govuk-heading-m">
          Cookies on Flood Map For Planning
        </h2>
        
        <div class="govuk-cookie-banner__content">
          
          <p class="govuk-body">We use analytics cookies to help understand how users use our service.</p>
          
        </div>
      </div>
    </div>

    
    <div class="govuk-button-group">
    
      
      <button type="button" class="govuk-button js-cookies-button-accept" data-module="govuk-button">
        Accept analytics cookies
      </button>
    
      
      <button type="button" class="govuk-button js-cookies-button-reject" data-module="govuk-button">
        Reject analytics cookies
      </button>
    
      
      <a class="govuk-link" href="cookies">View cookie preferences</a>
    
    </div>
    

  </div>
  
  <div class="govuk-cookie-banner__message js-cookies-accepted govuk-width-container" role="alert" hidden>

    <div class="govuk-grid-row">
      <div class="govuk-grid-column-two-thirds">
        
        <div class="govuk-cookie-banner__content">
          
          <p class="govuk-body">You've accepted analytics cookies. You can <a href="/cookies">change your cookie settings</a> at any time.</p>
          
        </div>
      </div>
    </div>

    
    <div class="govuk-button-group">
    
      
      <button type="button" class="govuk-button" data-module="govuk-button">
        Hide cookie message
      </button>
    
    </div>
    

  </div>
  
  <div class="govuk-cookie-banner__message js-cookies-rejected govuk-width-container" role="alert" hidden>

    <div class="govuk-grid-row">
      <div class="govuk-grid-column-two-thirds">
        
        <div class="govuk-cookie-banner__content">
          
          <p class="govuk-body">You've rejected analytics cookies. You can <a href="/cookies">change your cookie settings</a> at any time.</p>
          
        </div>
      </div>
    </div>

    
    <div class="govuk-button-group">
    
      
      <button type="button" class="govuk-button" data-module="govuk-button">
        Hide cookie message
      </button>
    
    </div>
    

  </div>
  
</div>




    
      <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>

    

    
  <header class="govuk-header" data-module="govuk-header">
  <div class="govuk-header__container govuk-width-container">
    <div class="govuk-header__logo">
      <a href="https://www.gov.uk/" class="govuk-header__link govuk-header__link--homepage">
        <svg
            focusable="false"
            role="img"
            xmlns="http://www.w3.org/2000/svg"
            viewBox="0 0 324 60"
            height="30"
            width="162"
            fill="currentcolor" class="govuk-header__logotype" aria-label="GOV.UK"
          ><title>GOV.UK</title>
            <g>
                <circle cx="20" cy="17.6" r="3.7"/>
                <circle cx="10.2" cy="23.5" r="3.7"/>
                <circle cx="3.7" cy="33.2" r="3.7"/>
                <circle cx="31.7" cy="30.6" r="3.7"/>
                <circle cx="43.3" cy="17.6" r="3.7"/>
                <circle cx="53.2" cy="23.5" r="3.7"/>
                <circle cx="59.7" cy="33.2" r="3.7"/>
                <circle cx="31.7" cy="30.6" r="3.7"/>
                <path d="M33.1,9.8c.2-.1.3-.3.5-.5l4.6,2.4v-6.8l-4.6,1.5c-.1-.2-.3-.3-.5-.5l1.9-5.9h-6.7l1.9,5.9c-.2.1-.3.3-.5.5l-4.6-1.5v6.8l4.6-2.4c.1.2.3.3.5.5l-2.6,8c-.9,2.8,1.2,5.7,4.1,5.7h0c3,0,5.1-2.9,4.1-5.7l-2.6-8ZM37,37.9s-3.4,3.8-4.1,6.1c2.2,0,4.2-.5,6.4-2.8l-.7,8.5c-2-2.8-4.4-4.1-5.7-3.8.1,3.1.5,6.7,5.8,7.2,3.7.3,6.7-1.5,7-3.8.4-2.6-2-4.3-3.7-1.6-1.4-4.5,2.4-6.1,4.9-3.2-1.9-4.5-1.8-7.7,2.4-10.9,3,4,2.6,7.3-1.2,11.1,2.4-1.3,6.2,0,4,4.6-1.2-2.8-3.7-2.2-4.2.2-.3,1.7.7,3.7,3,4.2,1.9.3,4.7-.9,7-5.9-1.3,0-2.4.7-3.9,1.7l2.4-8c.6,2.3,1.4,3.7,2.2,4.5.6-1.6.5-2.8,0-5.3l5,1.8c-2.6,3.6-5.2,8.7-7.3,17.5-7.4-1.1-15.7-1.7-24.5-1.7h0c-8.8,0-17.1.6-24.5,1.7-2.1-8.9-4.7-13.9-7.3-17.5l5-1.8c-.5,2.5-.6,3.7,0,5.3.8-.8,1.6-2.3,2.2-4.5l2.4,8c-1.5-1-2.6-1.7-3.9-1.7,2.3,5,5.2,6.2,7,5.9,2.3-.4,3.3-2.4,3-4.2-.5-2.4-3-3.1-4.2-.2-2.2-4.6,1.6-6,4-4.6-3.7-3.7-4.2-7.1-1.2-11.1,4.2,3.2,4.3,6.4,2.4,10.9,2.5-2.8,6.3-1.3,4.9,3.2-1.8-2.7-4.1-1-3.7,1.6.3,2.3,3.3,4.1,7,3.8,5.4-.5,5.7-4.2,5.8-7.2-1.3-.2-3.7,1-5.7,3.8l-.7-8.5c2.2,2.3,4.2,2.7,6.4,2.8-.7-2.3-4.1-6.1-4.1-6.1h10.6,0Z"/>
              </g>
            
              <circle class="govuk-logo-dot" cx="226" cy="36" r="7.3"/>
              <path d="M93.94 41.25c.4 1.81 1.2 3.21 2.21 4.62 1 1.4 2.21 2.41 3.61 3.21s3.21 1.2 5.22 1.2 3.61-.4 4.82-1c1.4-.6 2.41-1.4 3.21-2.41.8-1 1.4-2.01 1.61-3.01s.4-2.01.4-3.01v.14h-10.86v-7.02h20.07v24.08h-8.03v-5.56c-.6.8-1.38 1.61-2.19 2.41-.8.8-1.81 1.2-2.81 1.81-1 .4-2.21.8-3.41 1.2s-2.41.4-3.81.4a18.56 18.56 0 0 1-14.65-6.63c-1.6-2.01-3.01-4.41-3.81-7.02s-1.4-5.62-1.4-8.83.4-6.02 1.4-8.83a20.45 20.45 0 0 1 19.46-13.65c3.21 0 4.01.2 5.82.8 1.81.4 3.61 1.2 5.02 2.01 1.61.8 2.81 2.01 4.01 3.21s2.21 2.61 2.81 4.21l-7.63 4.41c-.4-1-1-1.81-1.61-2.61-.6-.8-1.4-1.4-2.21-2.01-.8-.6-1.81-1-2.81-1.4-1-.4-2.21-.4-3.61-.4-2.01 0-3.81.4-5.22 1.2-1.4.8-2.61 1.81-3.61 3.21s-1.61 2.81-2.21 4.62c-.4 1.81-.6 3.71-.6 5.42s.8 5.22.8 5.22Zm57.8-27.9c3.21 0 6.22.6 8.63 1.81 2.41 1.2 4.82 2.81 6.62 4.82S170.2 24.39 171 27s1.4 5.62 1.4 8.83-.4 6.02-1.4 8.83-2.41 5.02-4.01 7.02-4.01 3.61-6.62 4.82-5.42 1.81-8.63 1.81-6.22-.6-8.63-1.81-4.82-2.81-6.42-4.82-3.21-4.41-4.01-7.02-1.4-5.62-1.4-8.83.4-6.02 1.4-8.83 2.41-5.02 4.01-7.02 4.01-3.61 6.42-4.82 5.42-1.81 8.63-1.81Zm0 36.73c1.81 0 3.61-.4 5.02-1s2.61-1.81 3.61-3.01 1.81-2.81 2.21-4.41c.4-1.81.8-3.61.8-5.62 0-2.21-.2-4.21-.8-6.02s-1.2-3.21-2.21-4.62c-1-1.2-2.21-2.21-3.61-3.01s-3.21-1-5.02-1-3.61.4-5.02 1c-1.4.8-2.61 1.81-3.61 3.01s-1.81 2.81-2.21 4.62c-.4 1.81-.8 3.61-.8 5.62 0 2.41.2 4.21.8 6.02.4 1.81 1.2 3.21 2.21 4.41s2.21 2.21 3.61 3.01c1.4.8 3.21 1 5.02 1Zm36.32 7.96-12.24-44.15h9.83l8.43 32.77h.4l8.23-32.77h9.83L200.3 58.04h-12.24Zm74.14-7.96c2.18 0 3.51-.6 3.51-.6 1.2-.6 2.01-1 2.81-1.81s1.4-1.81 1.81-2.81a13 13 0 0 0 .8-4.01V13.9h8.63v28.15c0 2.41-.4 4.62-1.4 6.62-.8 2.01-2.21 3.61-3.61 5.02s-3.41 2.41-5.62 3.21-4.62 1.2-7.02 1.2-5.02-.4-7.02-1.2c-2.21-.8-4.01-1.81-5.62-3.21s-2.81-3.01-3.61-5.02-1.4-4.21-1.4-6.62V13.9h8.63v26.95c0 1.61.2 3.01.8 4.01.4 1.2 1.2 2.21 2.01 2.81.8.8 1.81 1.4 2.81 1.81 0 0 1.34.6 3.51.6Zm34.22-36.18v18.92l15.65-18.92h10.82l-15.03 17.32 16.03 26.83h-10.21l-11.44-20.21-5.62 6.22v13.99h-8.83V13.9"/>
            
          </svg>
        
      </a>
    </div>
  
    <div class="govuk-header__content">
    
      
      <a href="/" class="govuk-header__link govuk-header__service-name">
        Flood map for planning
      </a>
      
    
    
    </div>
  
  </div>
</header>



<div class="govuk-phase-banner govuk-width-container">
  <p class="govuk-phase-banner__content">
    <strong class="govuk-tag govuk-phase-banner__content__tag">
      Beta
    </strong>
    <span class="govuk-phase-banner__text">
      This is a new service. Help us improve it and <a class="govuk-link" href="/feedback">give your feedback</a>.
    </span>
  </p>
</div>




    
      <div class="govuk-width-container">
        
  
    <a href="javascript:history.back()" class="govuk-back-link">Back</a>

  

        <main class="govuk-main-wrapper" id="main-content">
          
   


  <div class="govuk-grid-row">
    <div class="govuk-grid-column-two-thirds">

      <h1 class="govuk-heading-xl">
        Accessibility statement for flood map for planning
      </h1>

      <p class="govuk-body">
        This accessibility statement applies to flood-map-for-planning.service.gov.uk.
      </p>

      <p class="govuk-body">
        This website is run by the Environment Agency. We want as many people as possible to be able to use this
        website. For example, that means you should be able to:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          change colours, contrast levels and fonts using browser or device settings
        </li>
        <li>
          zoom in up to 400% without the text spilling off the screen
        </li>
        <li>
          navigate most of the website using a keyboard or speech recognition software
        </li>
        <li>
          listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and
          VoiceOver)
        </li>
      </ul>

      <p class="govuk-body">
        We've also made the website text as simple as possible to understand.
      </p>

      <p class="govuk-body">
        <a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> has advice on making your device
        easier to use if you
        have a disability.
      </p>

      <h2 class="govuk-heading-l">
        How accessible this website is
      </h2>

      <p class="govuk-body">
        We know some parts of this website are not fully accessible:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          the mapping, and the tools that are part of it, are not fully accessible for those using assistive technology. Also
          in some situations the contrast of colours is not high enough to be legible to all users
        </li>
        <li>
          we link to guidance on GOV.UK that includes PDF documents that are not fully accessible to screen reader
          software
        </li>
        <li>
          the printable map that can be downloaded is in PDF form, so may not be fully accessible to screen reader
          software
        </li>
      </ul>

      <h2 class="govuk-heading-l">
        Feedback and contact information
      </h2>

      <p class="govuk-body">
        If you find any problems not listed on this page or think we're not meeting
        accessibility requirements, contact:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          email <a href="mailto:enquiries@environment-agency.gov.uk"
            class="govuk-link">enquiries@environment-agency.gov.uk</a>
        </li>
        <li>
          call 03708 506 506 (Monday to Friday, 8am to 5pm)
        </li>
      </ul>

      <p class="govuk-body">
        If you need information on this website in a different format like an accessible PDF, large print,
        easy read, audio recording or braille:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          email <a href="mailto:enquiries@environment-agency.gov.uk"
            class="govuk-link">enquiries@environment-agency.gov.uk</a>
        </li>
        <li>
          call 03708 506 506 (Monday to Friday, 8am to 5pm)
        </li>
      </ul>

      <h2 class="govuk-heading-l">
        Enforcement procedure
      </h2>

      <p class="govuk-body">
        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites
        and Mobile Applications) (No.2) Accessibility Regulations 2018 (the 'accessibility regulations').
      </p>

      <p class="govuk-body">
        If you're not happy with how we respond to your complaint
        <a href="https://www.equalityadvisoryservice.com/" class="govuk-link">
          contact the Equality Advisory and Support Service (EASS)
        </a>.
      </p>

      <h2 class="govuk-heading-l">
        Technical information about this website's accessibility
      </h2>

      <p class="govuk-body">
        The Environment Agency is committed to making its website accessible, in
        accordance with the Public Sector Bodies (Websites and Mobile Applications)
        (No. 2) Accessibility Regulations 2018.
      </p>

      <h2 class="govuk-heading-l">
        Compliance status
      </h2>

      <p class="govuk-body">
        The website has been tested against the Web Content Accessibility Guidelines (WCAG) 2.2 AA standard.
      </p>

      <p class="govuk-body">
        This website is partially compliant with the <a href="https://www.w3.org/TR/WCAG22/" class="govuk-link">Web
          Content Accessibility
          Guidelines version 2.2</a> AA standard, due to the non-compliances and exemptions listed below.
      </p>

      <h3 class="govuk-heading-m">
        Content that's not within the scope of the accessibility regulations
      </h3>

      <p class="govuk-body">
        The website has the following content which is out of scope of the accessibility regulations:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>online maps</li>
        <li>third party content which is out of our control, for example a corporate logo on the map the website uses</li>
        <li>PDF content that was published before 23 September 2018</li>
      </ul>

      <h3 class="govuk-heading-m">
        What we’re doing to improve accessibility
      </h3>

      <p class="govuk-body">
        We commit to providing alternative, non-digital access to the service where requests are made to the Environment
        Agency.
      </p>

      <p class="govuk-body">
        We commit to providing accessible PDFs where requests are made to the Environment Agency.
      </p>

      <h3 class="govuk-heading-m">
        Improvements to out of scope content
      </h3>

      <p class="govuk-body">
        Online maps are not within the scope of the accessibility regulations. However, we are committed to making our
        website as accessible as possible.
      </p>
      <p class="govuk-body">
        We are exploring ways to improve colour contrast on mapping.
      </p>
      <p class="govuk-body">
        We are exploring ways to improve how the website works for keyboard navigation and assistive technology.
      </p>
      <p class="govuk-body">
        We are also working to fix the following map issues found in our most recent accessibility test:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          on the interactive map the heading structure of the page does not make use of a logical structure,
          which means screen reader users may not be able to identify the structure and relationship of content
          (WCAG 2.2 success criterion 1.3.1, info and relationships)
        </li>
        <li>
          on the interactive map some heading text is styled as bold text, which means screen reader users cannot
          identify this as a heading (WCAG 2.2 success criterion 1.3.1, info and relationships)
        </li>
        <li>
          there is nothing to indicate to screen reader users that the draw a boundary content is interactive,
          or what its purpose is (WCAG 2.2 success criterion 1.3.1, info and relationships)
        </li>
        <li>
          on the interactive map colour was used as the only way of identifying keyboard focus in the listbox
          results when a user searches. This means that low vision users navigating the site using a keyboard
          may not be able to determine the focus on the page (WCAG 2.2 success criterion 1.4.1, use of colour)
        </li>
        <li>
          the interactive map relies on the difference in colour alone to indicate the difference in flood zones,
          which means it may not be possible for some users to tell them apart (WCAG 2.2 success criterion 1.4.1,
          use of colour)
        </li>
        <li>
          the ability to change the shape of a boundary is only available to users navigating the service using a
          mouse, which means users who cannot use a mouse to drag items are unable to edit the shape (WCAG 2.2
          success criterion 2.1.1, keyboard)
        </li>
        <li>
          on the interactive map, when selecting an option from the search listbox, users may not be able to
          cancel their selection once they have clicked the mouse. This means the list keeps closing and will
          not let the user select an option or scroll down the list without using the keyboard (WCAG 2.2 success
          criterion 2.5.2, pointer cancellation)
        </li>
        <li>
          the ability to change the shape of a boundary is only available to users navigating the service using a
          mouse, which means users who cannot use a mouse to drag items are unable to edit the shape (WCAG 2.2
          success criterion 2.5.7, dragging movements)
        </li>
        <li>
          screen reader users are not provided with instructions on how to navigate to the map content (WCAG 2.2
          success criterion 3.3.2, labels or instructions)
        </li>
        <li>
          there is nothing to indicate to screen reader users that the draw a boundary content is interactive,
          or what its purpose is (WCAG 2.2 success criterion 4.1.2, name, role, value)
        </li>
        <li>
          when adding a site boundary, there are multiple buttons on the page such as ‘Add’, ‘Edit’ and ‘Delete’
          that do not have descriptive accessible name to describe their purpose (WCAG 2.2 success criterion 2.4.6,
          headings and labels)
        </li>
        <li>
          content on the ‘Key’ panel of the map page could not be magnified without loss of information or
          functionality and without scrolling in 2 dimensions (WCAG 2.2 success criterion 1.4.10, reflow)
        </li>
        <li>
          on the ‘Key’ panel of the map page, the colour contrast of the orange ‘Flood defence’ symbol against the
          white background does not meet the threshold, which may be problematic for some visually impaired users
          (WCAG 2.2 success criterion 1.4.11, non-text contrast)
        </li>
        <li>
          the colour contrast of the overlapping data regions in the interactive map does not meet the threshold,
          which may be problematic for some visually impaired users (WCAG 2.2 success criterion 1.4.11, non-text contrast)
        </li>
        <li>
          the colour contrast of the black target selector against the dark blue ‘Flood zone 3’ region does not meet
          the threshold, which may be problematic for some visually impaired users (WCAG 2.2 success criterion 1.4.11,
          non-text contrast)
        </li>
      </ul>

      <h3 class="govuk-heading-m">
        Improvements made following accessibility testing
      </h3>

      <p class="govuk-body">
        We have fixed the following issues after our most recent accessibility test:
      </p>

      <ul class="govuk-list govuk-list--bullet">
        <li>
          visibly hidden content is still conveyed to screen reader users, which can be confusing (WCAG 2.2 success
          criterion 1.3.1, info and relationships)
        </li>
        <li>
          radio buttons have not been grouped appropriately, which does not give enough description for screen reader
          users to identify their purpose when out of context (WCAG 2.2 success criterion 1.3.1, info and relationships)
        </li>
        <li>
          the interactive map on mobile devices was highly problematic for screen reader users (WCAG 2.2 success criterion
          1.3.1, info and relationships)
        </li>
        <li>
          the interactive map on mobile devices was highly problematic for screen reader users (WCAG 2.2 success criterion
          2.1.1, keyboard)
        </li>
        <li>
          on the ‘Check your details’ page an image is provided to users so that they can see their selected location. The
          image receives keyboard focus, which may be disorientating for users navigating by keyboard alone (WCAG 2.2
          success criterion 2.4.3, focus order)
        </li>
        <li>
          on returning to the ‘Order your flood risk data’ page via the ‘Back’ link, the ‘Full name’ data previously
          entered by the user is not retained (WCAG 2.2 success criterion 3.3.7, redundant entry)
        </li>
        <li>
          on the ‘Check your details’ page an image is provided to users so that they can see their selected location. An
          alternative to the image is provided to screen reader users but this is not conveyed when using the NVDA screen
          reader (WCAG 2.2 success criterion 4.1.2, name, role, value)
        </li>
        <li>
          the ‘Hide this message’ button used in the cookie banner does not have a descriptive accessible name to describe
          its purpose (WCAG 2.2 success criterion 2.4.6, headings and labels)
        </li>
      </ul>

      <h2 class="govuk-heading-l">
        Preparation of this accessibility statement
      </h2>

      <p class="govuk-body">
        This statement was prepared on 30 March 2022. It was last reviewed on 2 May 2025.
      </p>

      <p class="govuk-body">
        This website was last tested against the WCAG 2.2 AA standard on 13 March 2025.
      </p>
      <p class="govuk-body">
        The test was carried out by the Digital Accessibility Centre (DAC).
      </p>
      <p class="govuk-body">
        They tested pages forming the most popular user journeys through the service using 2 processes:
      </p>
      <ul class="govuk-list govuk-list--bullet">
        <li>a manual technical audit using automated tools</li>
        <li>a dedicated team of user testers with differing disabilities test using a range of adaptive technologies.</li>
      </ul>
      <p class="govuk-body">
        <a href="/files/accessibility-report-2025-03.pdf" class="govuk-link" target="_blank">
          You can read the full accessibility test report (PDF).
        </a>
      </p>
    </div>
  </div>

        </main>
      </div>
    

    
  

<footer class="govuk-footer">
  <div class="govuk-width-container">
    

  
  <svg
    focusable="false"
    role="presentation"
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 64 60"
    height="30"
    width="32"
    fill="currentcolor" class="govuk-footer__crown"
  >
    <g>
        <circle cx="20" cy="17.6" r="3.7"/>
        <circle cx="10.2" cy="23.5" r="3.7"/>
        <circle cx="3.7" cy="33.2" r="3.7"/>
        <circle cx="31.7" cy="30.6" r="3.7"/>
        <circle cx="43.3" cy="17.6" r="3.7"/>
        <circle cx="53.2" cy="23.5" r="3.7"/>
        <circle cx="59.7" cy="33.2" r="3.7"/>
        <circle cx="31.7" cy="30.6" r="3.7"/>
        <path d="M33.1,9.8c.2-.1.3-.3.5-.5l4.6,2.4v-6.8l-4.6,1.5c-.1-.2-.3-.3-.5-.5l1.9-5.9h-6.7l1.9,5.9c-.2.1-.3.3-.5.5l-4.6-1.5v6.8l4.6-2.4c.1.2.3.3.5.5l-2.6,8c-.9,2.8,1.2,5.7,4.1,5.7h0c3,0,5.1-2.9,4.1-5.7l-2.6-8ZM37,37.9s-3.4,3.8-4.1,6.1c2.2,0,4.2-.5,6.4-2.8l-.7,8.5c-2-2.8-4.4-4.1-5.7-3.8.1,3.1.5,6.7,5.8,7.2,3.7.3,6.7-1.5,7-3.8.4-2.6-2-4.3-3.7-1.6-1.4-4.5,2.4-6.1,4.9-3.2-1.9-4.5-1.8-7.7,2.4-10.9,3,4,2.6,7.3-1.2,11.1,2.4-1.3,6.2,0,4,4.6-1.2-2.8-3.7-2.2-4.2.2-.3,1.7.7,3.7,3,4.2,1.9.3,4.7-.9,7-5.9-1.3,0-2.4.7-3.9,1.7l2.4-8c.6,2.3,1.4,3.7,2.2,4.5.6-1.6.5-2.8,0-5.3l5,1.8c-2.6,3.6-5.2,8.7-7.3,17.5-7.4-1.1-15.7-1.7-24.5-1.7h0c-8.8,0-17.1.6-24.5,1.7-2.1-8.9-4.7-13.9-7.3-17.5l5-1.8c-.5,2.5-.6,3.7,0,5.3.8-.8,1.6-2.3,2.2-4.5l2.4,8c-1.5-1-2.6-1.7-3.9-1.7,2.3,5,5.2,6.2,7,5.9,2.3-.4,3.3-2.4,3-4.2-.5-2.4-3-3.1-4.2-.2-2.2-4.6,1.6-6,4-4.6-3.7-3.7-4.2-7.1-1.2-11.1,4.2,3.2,4.3,6.4,2.4,10.9,2.5-2.8,6.3-1.3,4.9,3.2-1.8-2.7-4.1-1-3.7,1.6.3,2.3,3.3,4.1,7,3.8,5.4-.5,5.7-4.2,5.8-7.2-1.3-.2-3.7,1-5.7,3.8l-.7-8.5c2.2,2.3,4.2,2.7,6.4,2.8-.7-2.3-4.1-6.1-4.1-6.1h10.6,0Z"/>
      </g>
    
  </svg>

    
    
    <div class="govuk-footer__meta">
      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
        
        <h2 class="govuk-visually-hidden">Support links</h2>
        
        <ul class="govuk-footer__inline-list">
        
          <li class="govuk-footer__inline-list-item">
            <a class="govuk-footer__link" href="/cookies">
              Cookies
            </a>
          </li>
        
          <li class="govuk-footer__inline-list-item">
            <a class="govuk-footer__link" href="/accessibility-statement">
              Accessibility
            </a>
          </li>
        
          <li class="govuk-footer__inline-list-item">
            <a class="govuk-footer__link" href="/terms-and-conditions">
              Terms and conditions
            </a>
          </li>
        
          <li class="govuk-footer__inline-list-item">
            <a class="govuk-footer__link" href="/privacy-notice">
              Privacy notice
            </a>
          </li>
        
        </ul>
        
        
        <div class="govuk-footer__meta-custom">
          <p class="govuk-body-s">
              Built by the
              <a href="https://www.gov.uk/government/organisations/environment-agency">Environment Agency</a>
            </p>
            <p class="govuk-body-s" style="max-width: 70ch;">
              © Crown copyright and database rights 2025
              <a href = "https://www.ordnancesurvey.co.uk/">Ordnance Survey(OS)</a>
              AC0000807064. Use of the address and mapping data is subject to the
                <a href="/os-terms">Ordnance Survey terms and conditions</a>.
            </p>
        </div>
        
        
        <svg
          aria-hidden="true"
          focusable="false"
          class="govuk-footer__licence-logo"
          xmlns="http://www.w3.org/2000/svg"
          viewBox="0 0 483.2 195.7"
          height="17"
          width="41"
        >
          <path
            fill="currentColor"
            d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145"
          />
        </svg>
        <span class="govuk-footer__licence-description">
        
          All content is available under the
          <a
            class="govuk-footer__link"
            href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
            rel="license"
          >Open Government Licence v3.0</a>, except where otherwise stated
        
        </span>
      </div>
      <div class="govuk-footer__meta-item">
        <a
          class="govuk-footer__link govuk-footer__copyright-logo"
          href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/"
        >
        
          © Crown copyright
        
        </a>
      </div>
    </div>
  </div>
</footer>




    
  
  <script src="/assets/core.js"></script>
  <script type="module">
    import { initAll } from '/assets/govuk-frontend.min.js'
    initAll()
  </script>
  <script type="text/javascript">
    const analyticsAccount = 'replace_this'
    const analyticsAccepted = Boolean(document.cookie.match('GA=Accept'))

    if (analyticsAccount && analyticsAccepted) {
      const script = document.createElement('script')
      script.src = \`https://www.googletagmanager.com/gtm.js?id=\${analyticsAccount}\`
      script.onload = () => {
        window.dataLayer = window.dataLayer || []
        function gtag () { window.dataLayer.push(arguments) }
        gtag('consent', 'default', {
          ad_storage: 'granted',
          ad_personalization: 'granted',
          ad_user_data: 'granted',
          analytics_storage: 'granted'
        })
        window.dataLayer.push({
          'gtm.start': new Date().getTime(),
          event: 'gtm.js'
        })
      }
      document.body.appendChild(script)
    }
  </script>

  </body>
</html>
"
`;
